azure/durable_functions/models/DurableOrchestrationContext.py

```diff
@@ -224,9 +285,17 @@ def new_uuid(self) -> str:
         str
             New UUID that is safe for replay within an orchestration or operation.
         """
-        return new_uuid(context=self)
+        URL_NAMESPACE: str = "9e952958-5e33-4daf-827f-2fa12937b875"
```

Comment:
Did we move this out of a helper method, and if so, why?

Reply:
We did! We used to have a "utils"-like directory that contained all sorts of helpers. It **primarily** contained helpers for finding events in the history array. It also contained the `URL_NAMESPACE` and some deterministic-uuid helpers. 

After rewriting how iterate through the history, most of that directory became "dead code", leaving just the `new_uuid` helpers. Since so little remained, and things the new-uuid helpers are only used in one DF API, I figured it made more sense to just inline them in the method implementation itself.